Makefile: feature-flag command

brentguf opened the pull request:
# Changelog
<!-- Replace this comment by a bullet list. More info: https://www.notion.so/citizenlab/Changelog-How-it-works-f418426c75994454a332bf067634f3f1 -->

## Technical
- Makefile: `feature-toggle` command allows for `allowed` value (and gets renamed to `feature-flag`).

brentguf committed:
Extend feature-toggle to include the allowed value as well

adessy reviewed (approved):
It could be simplified a bit (especially the unused `enabled` and `allowed` variables), but OK to go! Are you sure people don't rely/use the existing `feature-toggle`?

brentguf committed:
Remove unused variables

brentguf commented:
> It could be simplified a bit (especially the unused enabled and allowed variables), but OK to go! 

@adessy Indeed, I've removed those variables. 

> Are you sure people don't rely/use the existing feature-toggle?

I've introduced it, so not even sure if others use it. That said, feature-flag can do the same. If it's troubling for others, we can add a default for allowed or something similar.
